variables.tf

```diff
@@ -241,3 +241,18 @@ variable "volume_storage_template" {
     default = ""
 }
 
+variable "upgrade_image" {
+    #OCP upgrade image
+    default = ""
+}
+
+variable "pause_time" {
```

Comment:
Since pause_time & delay_time is specific to upgrade.. rename them to upgrade_{pause,delay}_time.